ballerina-tests/tests/test.bal

```diff
@@ -1045,7 +1045,9 @@ function testHttpBlobInputOptional() returns error? {
 }
 
 
-@test:Config {}
+@test:Config {
+    enable: false
+}
```

Comment:
Shall we create an issue to enable these.

Reply:
Created an issue https://github.com/ballerina-platform/module-ballerinax-azure.functions/issues/660